grafana/Dockerfile-grafana

```
@@ -1,2 +1,7 @@
 FROM grafana/grafana:latest
-COPY --chown=grafana:grafana grafana.db /var/lib/grafana/grafana.db
+USER 0
```

Small readability nit.

```suggestion
USER root
```

---

got it